sycl/include/CL/sycl/exception.hpp

```diff
@@ -28,6 +28,8 @@ class context;
 /// \ingroup sycl_api
 class __SYCL_EXPORT exception : public std::exception {
 public:
+  __SYCL2020_DEPRECATED(
+      "SYCL 2020 does not have an exception constructor of 0 arguments.")
```

Comment:
```suggestion
      "The version of an exception constructor which takes no arguments is deprecated")
```
Maybe something like this?

Reply:
done